### PR TITLE
SUS-1202 | performance improvements for Chat right module

### PR DESCRIPTION
**Pull request by macbre**
https://wikia-inc.atlassian.net/browse/SUS-1202

Refresh user list for in-content chat widget only (it's cached by the MW parser).

The one on the right rail is always loaded fresh - the chat members list is always up to date as it comes with right rail lazy-loaded HTML.

**This makes the great majority of 1,4mm+ requests sent daily to `/wikia.php?controller=ChatRail&method=GetUsers&format=json` not needed**.

### CODE REVIEW

File: extensions/wikia/Chat2/js/ChatWidget.js [hunk 1]
```diff
@@ -16,13 +16,17 @@ var ChatWidget = {
 		}
 
 		// make sure we start processing after ChatModule templates is loaded
-		if ($('.chat-module').length) {
+		// SUS-1202: refresh user list for in-content chat widget, the one on the right rail is always loaded fresh
+		if ($('.chat-module').not('.rail-module').length) {
```
Comment (WikiaTech) [inline extensions/wikia/Chat2/js/ChatWidget.js]: MAJOR Merge this if statement with the nested one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by sqreek — APPROVED** [review]

**Review by mszabo-wikia — APPROVED** [review]

File: extensions/wikia/Chat2/js/ChatWidget.js (hunk 1 above)
Comment (WikiaTech) [inline extensions/wikia/Chat2/js/ChatWidget.js]: MAJOR Merge this if statement with the nested one. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**